api/views/canteen.py

```diff
@@ -1064,6 +1064,24 @@ def annotate_actions(queryset, year):
         # prep add diag action
         diagnostics = Diagnostic.objects.filter(canteen=OuterRef("pk"), year=year)
         user_canteens = user_canteens.annotate(diagnostic_for_year=Subquery(diagnostics.values("id")))
+        is_central_cuisine_query = Q(production_type=Canteen.ProductionType.CENTRAL) | Q(
+            production_type=Canteen.ProductionType.CENTRAL_SERVING
+        )
+        is_satellite_query = Q(production_type=Canteen.ProductionType.ON_SITE_CENTRAL)
+        incomplete_canteen_data_query = (
+            Q(yearly_meal_count=None)
+            | Q(daily_meal_count=None)
+            | Q(siret=None)
+            | Q(name=None)
+            | Q(city_insee_code=None)
+            | Q(production_type=None)
+            | Q(management_type=None)
```

Comment:
Need a check for economic model too

Reply:
Thanks :)  Fix: https://github.com/betagouv/ma-cantine/pull/2312/commits/b292f7d1fac61ed9b82aff112b5994100e89ac2a